examples/src/test/scala/docs/KafkaToSlickExample.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+package docs
+
+import scala.concurrent.duration._
+
+import akka.Done
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.Behavior
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.scaladsl.adapter._
+import akka.kafka.ConsumerSettings
+import akka.kafka.ProducerSettings
+import akka.kafka.Subscriptions
+import akka.kafka.scaladsl.SendProducer
+import akka.projection.ProjectionId
+import akka.projection.internal.MergeableOffsets
+import akka.projection.kafka.KafkaSourceProvider
+import akka.projection.scaladsl.SourceProvider
+import akka.projection.slick.SlickProjection
+import com.typesafe.config.ConfigFactory
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.serialization.ByteArrayDeserializer
+import org.apache.kafka.common.serialization.ByteArraySerializer
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.apache.kafka.common.serialization.StringSerializer
+import slick.basic.DatabaseConfig
+import slick.dbio.DBIOAction
+import slick.jdbc.H2Profile
+
+object KafkaToSlickExample {
+
+  ActorSystem[Nothing](Guardian(), "KafkaToSlickExample")
+
+  object Guardian {
+    def apply(): Behavior[Nothing] = {
+      Behaviors.setup[Nothing] { context =>
+        implicit val system = context.system
+
+        val config = ConfigFactory.parseString("""
+        akka {
+         loglevel = "DEBUG"
+         projection.slick = {
+  
+           profile = "slick.jdbc.H2Profile$"
+  
+           # TODO: configure connection pool and slick async executor
+           db = {
+            url = "jdbc:h2:mem:test1"
+            driver = org.h2.Driver
+            connectionPool = disabled
+            keepAliveConnection = true
+           }
+         }
+        }
+        """)
```

Comment:
config should be for the ActorSystem, and here accessed via `context.system.settings.config`

Reply:
Do you mean move this to an `application.conf` and load it from the ActorSystem?  How would we provide configuration for other examples in this project if it's in a file? Should we create a sub project per example?

Reply:
It can still be in code, but just defined above when the ActorSystem is created.

Reply:
Ok, got it.

TODO

Reply:
I think you meant that I should extend `ScalaTestWithActorTestKit` like the other integration tests, but I couldn't do this since I'm using Alpakka Kafka's testkit.  I had to recreate some of the boilerplate, but the config is applied in the same way as `SlickSpec`.  WDYT?